src/app/common/utils/system-notifications/system-notifications.service.ts

```diff
@@ -11,9 +11,10 @@ export class SystemNotificationsService {
     private readonly snackBar: MatSnackBar,
   ) { }
 
-  public showNotification({ message, dismissBtnText = 'OK' }: IShowNotificationConfig): void {
+  public showNotification({ message, dismissBtnText = 'OK', type="success" }: IShowNotificationConfig): void {
```

Comment:
On those two lines that you have implemented you should see in your editor two linter issues/errors that you need to fix -> if you don't see them you should configure your editor properly :) You can also run `npm run lint` which will tell you about those linter issues. For example here you should not use double quotes.